### PR TITLE
Optimise volume creation for vol-attach-tests

### PR DESCRIPTION
**Pull request by ambiknai**

### CODE REVIEW

File: tests/e2e/volume_attach.go [hunk 1]
```diff
@@ -354,7 +312,18 @@ var _ = Describe("[ics-e2e] [volume-attachment-limit] [default] [12-volumes]", f
...
+	It("Verify volume attachment without any change in configmap", func() {
+		By("DEFAULT VOLUME ATTACHMENT WITH 12 VOLUMES")
+		CreateStorageClass(customSCName, cs)
+		// Defer the deletion of the StorageClass object.
+		defer func() {
+			if err := cs.StorageV1().StorageClasses().Delete(context.Background(), customSCName, metav1.DeleteOptions{}); err != nil {
+				panic(err)
+			}
+		}()
+		for i := 0; i < 13; i++ {
```
Comment (GunaKKIBM) [inline tests/e2e/volume_attach.go:324]: Did we delete volumes from pvc-0 to pvc-3 anywhere earlier? Also, conisdering that these were created earlier, can range from 4 to 13?

File: tests/e2e/volume_attach.go [hunk 2]
```diff
@@ -966,3 +776,63 @@ func UpdateVolumeAttachmentLimit(client clientset.Interface, limit string) bool
...
+			Namespace: namespace,
+		},
+		Spec: corev1.PersistentVolumeClaimSpec{
+			StorageClassName: &customSCName,
+			AccessModes: []corev1.PersistentVolumeAccessMode{
+				corev1.ReadWriteOnce,
+			},
+			Resources: corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceName(corev1.ResourceStorage): resource.MustParse("10Gi"),
```
Comment (GunaKKIBM) [inline tests/e2e/volume_attach.go:794]: Can we reduce this to 5Gi? might help reduce the billing

**Review by GunaKKIBM — APPROVED** [review]
lgtm

**Comment by prankulmahajan** [comment]
@ambiknai one small comment, as of now the msgs are shown like this
```
VPC-BLK-CSI-TEST: DEFAULT VOLUME ATTACHMENT WITH 13 VOLUMES: PASS
VPC-BLK-CSI-TEST: DEFAULT VOLUME ATTACHMENT WITH 12 VOLUMES: PASS
VPC-BLK-CSI-TEST: VOLUME_ATTACHMENT_LIMIT SET TO 3 && ATTACHED VOLUMES 3: PASS
VPC-BLK-CSI-TEST: VOLUME_ATTACHMENT_LIMIT SET TO 3 && ATTACHED VOLUMES 4: PASS
```

As per @arahamad suggestion can we update it to,
```
VPC-BLK-CSI-TEST: DEFAULT VOLUME ATTACHMENT WITH 13 VOLUMES failure: PASS
VPC-BLK-CSI-TEST: DEFAULT VOLUME ATTACHMENT WITH 12 VOLUMES: PASS
VPC-BLK-CSI-TEST: VOLUME_ATTACHMENT_LIMIT SET TO 3 && ATTACHED VOLUMES 3: PASS
VPC-BLK-CSI-TEST: VOLUME_ATTACHMENT_LIMIT SET TO 3 && ATTACHED VOLUMES 4 failure: PASS
```

**Review by prankulmahajan — APPROVED** [review]
lgtm

**Pull request merged**